modules/schema/src/main/java/org/apache/ignite/internal/schema/configuration/SchemaConfigurationConverter.java

```diff
@@ -201,12 +199,12 @@ public static TableIndex convert(TableIndexView idxView) {
             case SORTED_TYPE:
                 boolean sortedUniq = idxView.uniq();
 
-                SortedMap<Integer, SortedIndexColumn> sortedCols = new TreeMap<>();
-                
+                SortedMap<String, SortedIndexColumn> sortedCols = new TreeMap<>();
```

Comment:
Here we used TreeMap to keep order with indexIdx or ColumnIdx keys. But for String keys (if configuration will keep order) it enough to use just List.